app/src/main/res/values-night/themes.xml

```diff
@@ -10,7 +10,9 @@
         <item name="colorSecondaryVariant">@color/teal_200</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
+        <item name="android:statusBarColor" >?attr/colorPrimaryVariant</item>
```

Comment:
There's an extra space between
"android:statusBarColor"
and
\>?attr/colorPrimaryVariant

Reply:
Done